src/Logic/CollectionsHelper.php

```diff
@@ -0,0 +1,346 @@
+<?php
+
+namespace Newspack\MigrationTools\Logic;
+
+use Exception;
+use Newspack\MigrationTools\Traits\Has_Unique_Identifier;
+use Newspack\Optional_Modules\Collections;
+use WP_Error;
+use WP_Term;
+
+/**
+ * Collections Helper class.
+ */
+class CollectionsHelper {
+	use Has_Unique_Identifier;
+
+	public const UNIQUE_COLLECTION_IDENTIFIER_META_KEY          = '_nmt_collection_uniqid';
+	public const UNIQUE_COLLECTION_SECTION_IDENTIFIER_META_KEY  = '_nmt_collection_section_uniqid';
+	public const UNIQUE_COLLECTION_CATEGORY_IDENTIFIER_META_KEY = '_nmt_collection_category_uniqid';
+
+	/**
+	 * Collection metadata map.
+	 */
+	const COLLECTION_META_MAP = [
+		'thumbnail'      => '_thumbnail_id',
```

Comment:
Very minor but I like "_id" when it's an integer.  So maybe:
```suggestion
		'thumbnail_id'      => '_thumbnail_id',
```

Reply:
Agreed and changed!